eqcorrscan/tests/clustering_test.py

```diff
@@ -84,10 +84,32 @@ def test_distance_matrix(self):
         for st in stream_list:
             for tr in st:
                 tr.data = tr.data[0:shortest_tr]
-        dist_mat = distance_matrix(stream_list=stream_list, cores=4)
+        # Test with zero shift-length
+        dist_mat, shift_mat = distance_matrix(stream_list=stream_list, cores=4)
         self.assertEqual(dist_mat.shape[0], len(stream_list))
         self.assertEqual(dist_mat.shape[1], len(stream_list))
 
+        # Test with some shift-length, but only shift templates as a whole
```

Comment:
Can you make this a separate test please? It will help with debugging, and will allow it to run concurrently to other tests.